pandas/io/stata.py

```diff
@@ -3216,10 +3225,14 @@ class StataWriter118(StataWriter117):
         Smaller columns can be converted by including the column name.  Using
         StrLs can reduce output file size when strings are longer than 8
         characters, and either frequently repeated or sparse.
+    version : int, optional
```

Comment:
can you add a versionadded tag

Reply:
I don't think the versionadded is needed here since the entire class is new in 1.0.0, and the class is versionadded: 1.0.0

Here: 

https://github.com/pandas-dev/pandas/pull/30959/files/287ca23592e44b5ab82069e23a1e2a3b191ce9e3#diff-2def41db3b4d04fb64de8313c0e6c25eL3184